xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1171 @@
+      with the hostname of the webserver providing the Helm chart repository on
+      the internal network.
+     </para>
+<screen>&prompt.user;<command>mkdir <replaceable>/tmp/charts</replaceable></command></screen>
+<screen>&prompt.user;<command>cd <replaceable>/tmp/charts</replaceable></command></screen>
+<screen>&prompt.user;<command>helm-mirror --new-root-url <replaceable>http://mymirror.local/charts</replaceable> https://kubernetes-charts.suse.com <replaceable>/tmp/charts</replaceable></command></screen>
+   </step>
+   <step>
+    <para>
+     Translate the chart information into the <literal>skopeo</literal> format.
+    </para>
+<screen>&prompt.user;<command>mkdir <replaceable>/tmp/skopeodata</replaceable></command></screen>
+<screen>&prompt.user;<command>helm-mirror inspect-images <replaceable>/tmp/charts</replaceable> -o skopeo --file-name <replaceable>sync.yaml</replaceable></command></screen>
+   </step>
+   <step>
+    <para>
+     Download all the referenced images using <command>skopeo</command>.
+    </para>
+<screen>&prompt.user;<command>cd /tmp/skopeodata/</command></screen>
+<screen>&prompt.user;<command>skopeo sync --source-yaml <replaceable>sync.yaml</replaceable> dir:<replaceable>/tmp/skopeodata</replaceable></command></screen>
+    <para>
+     <command>skopeo</command> will automatically create a directory named
+     after the hostname of the registry from which you are downloading the
+     images. The final path will be something like <filename>/tmp/skopeodata/registry.suse.com/</filename>.
+    </para>
+   </step>
+   <step>
+    <para>
+     Populate the local registry with the downloaded data.
+    </para>
+    <para>
+     For <literal>--dest-creds</literal> you must use the credentials you
+     created during <xref linkend="sec.deploy.scenarios.airgap.container-registry.mirror" />.
+    </para>
+<screen>&prompt.user;<command>skopeo sync --dest-creds <replaceable>USERNAME:PASSWORD</replaceable> \
+dir:<replaceable>/tmp/skopeodata/registry.suse.com/</replaceable> <replaceable>docker://mymirror.local:5000</replaceable></command>
+     </screen>
+   </step>
+   <step>
+    <para>
+     After the synchronization is done, you can remove the <filename>skopeodata</filename>
+     directory.
+    </para>
+<screen>&prompt.user;<command>rm -rf /tmp/skopeodata</command></screen>
+   </step>
+  </procedure>
+
+   <procedure>
+    <title>Transfer Data To Secure Storage</title>
+    <step>
+     <para>
+      Connect the trusted storage to the external mirror.
+     </para>
+    </step>
+    <step>
+     <para>
+      Transfer the container image data to the trusted storage. This will remove
+      all files and directories that are no longer present on the external host
+      from the trusted storage.
+     </para>
+<screen>&prompt.user;<command>rsync -aP /tmp/skopeodata/registry.suse.com/ <replaceable>/mnt/storage/registry/</replaceable> --delete</command>
```

Comment:
**DANGER:** this command will fail. We have to rsync the registry storage data to the USB disk, not the `/tmp/skopeodata` (which has just been removed - as it was supposed to happen).

The right command is going to be:

```
<screen>&prompt.user;<command>rsync -aP <replaceable>/var/lib/registry/</replaceable> <replaceable>/mnt/storage/registry/</replaceable> --delete</command>
```